### PR TITLE
Allow phpunit 9

### PR DESCRIPTION
**Pull request by acbramley**

### CODE REVIEW

**Comment by acbramley** [comment]
Looks like we'll need a new major for 9.

**Comment by larowlan** [comment]
:+1:

**Comment by kimpepper** [comment]
`src/Util/Configuration` has been removed from PHPUnit 9 so this might need a bit of effort to work out the required changes.

File: src/FinderCommand.php [hunk 1]
```diff
@@ -33,19 +36,45 @@ protected function execute(InputInterface $input, OutputInterface $output) {
...
+      foreach ($testSuites as $suite) {
+        $suite = $config->getTestSuiteConfiguration($suite);
+        foreach (new \RecursiveIteratorIterator($suite->getIterator()) as $test) {
+          if ($test instanceof TestCase) {
+            $testFilenames[] = ((new \ReflectionClass($test))->getFileName());
+          }
+        }
+      }
+
+    }
```
Comment (acbramley) [inline src/FinderCommand.php:49]: IMO we should just have a new major?

Comment (kimpepper) [reply src/FinderCommand.php:49]: Yeah, I agree on a new major.

**Review by kimpepper — APPROVED** [review]

**Pull request merged**